en/docs/develop/apis/identity-governance-rest-api.md

```diff
@@ -15,112 +15,86 @@ and properties of connectors as mentioned below. <br>
+    | Temporary password generation extension class  | EmailVerification.AskPassword.PasswordGenerator  | Temporary password generation extension point in ask password feature |
+    | Manage JIT provisioning purposes  | _url_listPurposeJITProvisioning  | Click here to manage just in time provisioning purposes. |
 
-??? "Account Management Policies"
+??? "Login Attempts Security"
 
-    **Lock Idle Accounts**
-    
-    | Property display name | Property name | Purpose |
-    | ------- | -------|---|
-    | Enable   | suspension.notification.enable   |  |
-    | Lock Account After (days)   | suspension.notification.account.disable.delay   |  |
-    | Alert Users before account locking in days (comma-separated list)   | suspension.notification.delays   |  |
-    
-    **Account Recovery**
+    **Account Lock**
     
     | Property display name | Property name | Purpose |
     | ------- | -------|---|
-    | Enable Notification Based Password Recovery   | Recovery.Notification.Password.Enable   |  |
-    | Enable reCaptcha for Password Recovery   | Recovery.ReCaptcha.Password.Enable   |  |
-    | Enable Security Question Based Password Recovery   | Recovery.Question.Password.Enable   |  |
-    | Number Of Questions Required For Password Recovery   | Recovery.Question.Password.MinAnswers   |  |
-    | Enable reCaptcha for Security Questions Based Password Recovery   | Recovery.Question.Password.ReCaptcha.Enable   | Show captcha for challenge question based password recovery |
-    | Max Failed Attempts for ReCaptcha   | Recovery.Question.Password.ReCaptcha.MaxFailedAttempts   |  |
-    | Enable Username Recovery   | Recovery.Notification.Username.Enable   |  |
-    | Enable reCaptcha for Username Recovery   |  Recovery.ReCaptcha.Username.Enable  |  |
-    | Enable Internal Notification Management   | Recovery.Notification.InternallyManage  | Set to false if the client application handles notification sending |
-    | Notify when Recovery Success   | Recovery.NotifySuccess   |  |
-    | Notify when Questions Based Recovery Starts   | Recovery.Question.Password.NotifyStart   |  |
-    | Recovery Link Expiry Time   | Recovery.ExpiryTime   |  |
-    | Enable forced challenge questions   | Recovery.Question.Password.Forced.Enable   | Force users to provide answers to challenge questions during sign in |
-    | Recovery callback URL regex   | Recovery.CallbackRegex   |  |
-    
-    **User Self Registration**
+    | Lock user accounts   | account.lock.handler.enable   | Lock user accounts on failed login attempts |
+    | Maximum failed login attempts   | account.lock.handler.On.Failure.Max.Attempts   | Number of failed login attempts allowed until account lock. |
+    | Initial account lock duration   | account.lock.handler.Time   | Initial account lock time period in minutes. Account will be automatically unlocked after this time period. |
```

Comment:
Initial account lock time period in minutes.

This doesn't seem self explanatory.